File: backend2/DataAccess/DaoGenerated/システム共有/tt_afrenrakusakiDto.cs
// *******************************************************************
// 業務名称　: 互助防疫システム
// 機能概要　: DTO定義
//             連絡先テーブル
// 作成日　　: 2024.07.23
// 作成者　　: 
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.DataAccess
{
    public class tt_afrenrakusakiDto : DaEntityModelBase
    {
		public const string TABLE_NAME = "tt_afrenrakusaki";
		public string atenano { get; set; }                       //宛名番号
		public string jigyocd { get; set; }                       //個人連絡先利用事業コード
		public string? tel { get; set; }                          //電話番号
		public string? keitaitel { get; set; }                    //携帯番号
		public string? emailadrs { get; set; }                    //E-mailアドレス
		public string? emailadrs2 { get; set; }                   //E-mailアドレス2
		public string? syosai { get; set; }                       //連絡先詳細
		public string? regsisyo { get; set; }                     //登録支所
		public string reguserid { get; set; }                     //登録ユーザーID
		public DateTime regdttm { get; set; }                     //登録日時
		public string upduserid { get; set; }                     //更新ユーザーID
		public DateTime upddttm { get; set; }                     //更新日時
    }
}

File: backend/BusinessService/Db/DaoGenerated/システム共有/tt_afrenrakusakiDto.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: DTO定義
'             連絡先テーブル
' 作成日　　: 2024.07.23
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace JBD.GJS.Db
    Public Class tt_afrenrakusakiDto
        Inherits DaEntityModelBase
        Public Const TABLE_NAME As String = "tt_afrenrakusaki"
        Public Property atenano As String                       '宛名番号
        Public Property jigyocd As String                       '個人連絡先利用事業コード
        Public Property tel As String                          '電話番号
        Public Property keitaitel As String                    '携帯番号
        Public Property emailadrs As String                    'E-mailアドレス
        Public Property emailadrs2 As String                   'E-mailアドレス2
        Public Property syosai As String                       '連絡先詳細
        Public Property regsisyo As String                     '登録支所
        Public Property reguserid As String                     '登録ユーザーID
        Public Property regdttm As Date                     '登録日時
        Public Property upduserid As String                     '更新ユーザーID
        Public Property upddttm As Date                     '更新日時
    End Class
End Namespace
